### PR TITLE
[doc] Spec NTFS-FR : adding lines and networks

### PR DESCRIPTION
**Pull request by prhod**
Here is the addition of Lines and Networks in the specs.
There is a  very few properties because they are all optional. we'll add them later if requested.

### CODE REVIEW

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ A NTFS feed is not sufficient to generate a valid Netex Profil France feed:
 - `ParticipantRef` : The ParticipantRef code must be provided,
 - `StopProviderCode` : The code used to identify the provider in the stop_id generation.
 
+## Id of objects
+
+According to the Netex Specification, object identifiers should have a specific formating.
```
Comment (datanel) [inline src/documentation/ntfs_to_netex_france_specs.md]: not sure
```suggestion
According to the Netex Specification, object identifiers should have a specific formatting.
```

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ A NTFS feed is not sufficient to generate a valid Netex Profil France feed:
...
+The formating for stops is describe in the [arrets.xml](#arretsxml) part of this specification.
```
Comment (datanel) [inline src/documentation/ntfs_to_netex_france_specs.md]:
```suggestion
The formatting for stops is described in the [arrets.xml](#arretsxml) part of this specification.
```

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ A NTFS feed is not sufficient to generate a valid Netex Profil France feed:
...
+For the other objects, the following format is used: **FR:[object type]:[object id]:**
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: The `object_id` might create problems if it contains `:`. We need to replace all `:` (with `_` I suppose, like for stops' identifiers?).

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 4]
```diff
@@ -140,3 +162,47 @@ warnings:
...
+## lignes.xml
+
+### Top level structure
+
+Example: 
+
+>   <?xml version="1.0" encoding="UTF-8"?>
+    <CompositeFrame 
+            id="FR:CompositeFrame:NETEX-20190709T093214Z:LOC" 
+            version="1.8">
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: Probably another version?
```suggestion
            version="any">
```

Comment (woshilapin) [reply src/documentation/ntfs_to_netex_france_specs.md]: We'll leave it as it is for now. Following what we have in NeTEx IDF at the moment.

**Pull request merged**